### ISSUE TITLE
Get metadata on sources

### ISSUE THREAD

**Issue by fredmny** [fal-ai/fal#506]
**Context**
To use fal-ai to notify users via slack on failed source tests we would like to use the owner configured within the `metadata` tag, the same way as it's possible for models

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
To have a metadata parameter for each source (callable via list_sources() ) the same way it's already possible with models

**Describe alternatives you've considered**
The only possible, but not viable, alternative I could think of, would be to hardcode it into the script based on the model name

**Additional context**

**Is there an existing feature request for this?**
 * [x]  I have searched the existing issues

**Comment by chamini2** [comment]
Hello, @fredmny! Thanks for the idea. I will take a look at this now.

**Comment by chamini2** [comment]
And this has been released! Please test it with [v0.5.2](https://github.com/fal-ai/fal/releases/tag/v0.5.2)

*****

I guess it would look something like this:

```py
from fal.typing import *

for source in list_sources():
  if any(t.status != 'success' for t in source.tests):
    send_message(f"Test for {source.name},{source.table_name} failed, {source.meta.get('owner', '<no owner found>')}")
```

Are you currently using `fal flow run`, `fal run` or the `FalDbt` object?

**Comment by fredmny** [comment]
Hey, sorry for the late reply!!! I was getting ready for vacation and didn't see it. Thanks too much!
So far we are using the `fal run` command
